### PR TITLE
Build amd64/arm64 multiarch images

### PR DESCRIPTION
**Pull request by frebib**
Signed-off-by: Adam Dodman <adam@adam-ant.co.uk>
Signed-off-by: Joe Groocock <me@frebib.net>

### CODE REVIEW

File: .drone.star [hunk 1]
```diff
@@ -27,23 +50,54 @@ def step(alpinever,tags=[]):
         "pull": "always",
         "settings": {
           "run": vertest + "su-exec nobody apk --version",
+          "repo": repo,
```
Comment (Adam-Ant) [inline .drone.star]: Overwriting the repo variable from the global scope seems kinda cludgy, could cause some weirdness if the local-scope variable ever doesn't work.

File: .drone.star [hunk 2]
```diff
@@ -1,24 +1,47 @@
...
+      if ver == versions[-1]:
+        tags.append("latest")
+      builds.append(publish(ver, srctpl, depends_on, tags))
 
-def step(alpinever,tags=[]):
-  vertest = "grep -q '%s' /etc/alpine-release && " % alpinever if alpinever != "edge" else ""
+  return builds
+
+def step(ver, arch, key, repo):
+  vertest = "grep -q '%s' /etc/alpine-release && " % ver if ver != "edge" else ""
```
Comment (Adam-Ant) [inline .drone.star:27]: Do we even need this any more?

**Review by Adam-Ant — COMMENTED** [review]
Few nitpicks, but generally LGTM

File: .drone.star (hunk 1 above)
Comment (frebib) [reply .drone.star]: It's probably well-defined, but I'll rename for clarity

File: .drone.star (hunk 2 above)
Comment (frebib) [reply .drone.star:27]: Let's find out

Comment (frebib) [reply .drone.star:27]:
```
root@1b79854e629e /> grep -q edge /etc/alpine-release
root@1b79854e629e /> echo $?
1
root@1b79854e629e /> cat /etc/alpine-release
3.16.0_alpha20220328
```

Comment (Adam-Ant) [reply .drone.star:27]: ah, so edge doesn't have a release file. That tracks.

**Pull request merged**